tests/cupy_tests/manipulation_tests/test_dims.py

```diff
@@ -268,6 +258,12 @@ def test_broadcast(self):
         self.assertEqual(broadcast_np.size, broadcast_cp.size)
         self.assertEqual(broadcast_np.nd, broadcast_cp.nd)
 
+    @testing.numpy_cupy_array_list_equal()
+    def test_broadcast_arrays(self, xp):
```

Comment:
Would you perform this test for all dtypes because the change seems to have degradation?